src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementBackedDocumentsService.java

```diff
@@ -94,28 +101,30 @@ private Claim getClaimByExternalId(String externalId, String authorisation) {
         return claimService.getClaimByExternalId(externalId, authorisation);
     }
 
-    private DocumentDetails uploadToDocumentManagement(
+    @Override
+    public byte[] uploadToDocumentManagement(
```

Comment:
returned bytes is not modified, just returned without change, does it need to have return